config/locales/simple_form.en.yml

```diff
@@ -2,6 +2,7 @@ en:
   simple_form:
     "yes": 'Yes'
     "no": 'No'
+    cancel: 'Cancel'
```

Comment:
Can you add the corresponding `es` translations.  Using google translate would be good enough for a first pass